frontend/src/components/Tree/Tree.jsx

```diff
@@ -8,12 +8,12 @@ import "./Tree.module.css";
  * This function takes node-text grouping, and inter-node link grouping and performs the animation
  * of the links between them.
  * @param nodeGroupEnter : A grouping of nodes and text boxes grouped together under the SVG tag
- * @param enteringAndUpdatingLinks : The inter-node links, represented as lines joining 
+ * @param enteringAndUpdatingLinks : The inter-node links, represented as lines joining
  * nodes together on the tree.
  */
 
-function animateTree(nodeGroupEnter, enteringAndUpdatingLinks) {
-  nodeGroupEnter
+function animateTree(nodeGroupEnterAndUpdate, enteringAndUpdatingLinks) {
+  nodeGroupEnterAndUpdate
```

Comment:
Is the changes to this file the same as your other MR?

Reply:
Yup, I had just merged that branch into this one

Reply:
I will probably have to deal with a good number of merge conflicts when merging with your branch 😂

Reply:
:sob: